import-upstream

```diff
@@ -63,6 +63,25 @@ def main() -> int:
 
     os.chdir(args.repo)
 
+    tag = f"upstream/{args.version.version}"
+    print(f"checking for pre-existing '{tag}' tag", end="... ", flush=True)
+    try:
+        out = subprocess.check_output(
+            ('git', 'tag', '--list', tag),
+            stderr=subprocess.STDOUT,
+        )
+    except subprocess.CalledProcessError as cpe:
+        print("check crashed, exiting.")
+        out = cpe.output.decode().strip() if cpe.output else ''
+        if out:
+            print(out)
+        return cpe.returncode
+    else:
+        if tag in out.decode():
```

Comment:
This is enough because the `git tag --list QUERY` has no wildcards. It will either match the tag verbatim or nothing.